src/utils.ts

```diff
@@ -0,0 +1,5 @@
+function sleep(ms: number) {
```

Comment:
I see this function in two places: in `src/utils.ts` (this file) and in `meilisearch-test-utils.ts`.
Can you import this function from this file (`src/utils.ts`) in `meilisearch-test-utils.ts` to avoid duplicated code? 🙂

Reply:
So after talking with Clementine. She accepts that I leave it like this. Because I do want to separate my project code from my testing code. 
The only link between them should be the import. If tomorrow I want to try out my project some other way I just have to change the import. The tests run on their own without needing not-exported-material from the SDK.